public/app/plugins/datasource/grafana-azure-monitor-datasource/components/metrics.ts

```diff
@@ -20,6 +21,7 @@ export function useMetricsMetadata(
     aggOptions: [],
     timeGrains: [],
     dimensions: [],
+    isLoading: true,
```

Comment:
I think this should just be false, and it's set to true when we actually make an API call. Now, when you first make a panel Aggregations will load until you fill out all the previous inputs.